gateway/exporters/statsd/statsd.go

```diff
@@ -27,6 +27,8 @@ type Metric struct {
 	Measurement string            `json:"Metric"`
 	Namespace   string            `json:"Namespace"`
 	Dims        map[string]string `json:"Dims"`
+	// ns since epoch
+	Timestamp   int64			  `json:"Timestamp"`
```

Comment:
do we require timestamps in the JSON as well?

we should check that it doesn't impact the metric processing on the mdm agent's side